playground/LocaleUtils/index.js

```diff
@@ -24,7 +24,7 @@ export async function assertNoEnglishLeaks(mockFile, viewText, noTranslationCont
     Handle untranslated colon at the end of Odyssey screenreader text for field-level errors (reads as 'Error:')
     TODO: Ask Odyssey why colon is not part of translation bundle
   */
-  extractedString = extractedString.replace('《:', '《');
+  extractedString = extractedString.replace(/《:/g, '《');
```

Comment:
Fixes false-negative test result for `enroll-profile-with-password-returns-multiple-errors` for Gen3

Reply:
There are 2 occurrences of "Error:" visually hidden text:

<img width="473" alt="Screenshot 2024-04-15 at 16 41 02" src="https://github.com/okta/okta-signin-widget/assets/72614880/7ccb42f5-2a33-4337-86cf-0bee5f1dfa76">

<img width="931" alt="Screenshot 2024-04-15 at 16 40 48" src="https://github.com/okta/okta-signin-widget/assets/72614880/ba2fa41a-49a8-4517-81df-877af678c1b5">

https://github.com/okta/odyssey/blob/main/packages/odyssey-react-mui/src/FieldError.tsx#L47